cmd/zdb/zdb.c

```diff
@@ -6751,9 +6753,12 @@ dump_block_stats(spa_t *spa)
 	if (leaks)
 		return (2);
```

Comment:
Looks like your missing a `umem_free()` here.

Reply:
Thank you for picking that up, I also fixed https://github.com/openzfs/zfs/pull/13807/commits/f9687106ff291b2cef65d6a3da97bd09a775b739#diff-0f5313b317e4f72623f33a7c61d0e72115913737e19d910b093dbc7359cc9681R6547